code/game/gamemodes/wizard/soulstone.dm

```diff
@@ -350,15 +325,21 @@
 	S.key = T.key
 	if(U)
 		S.faction |= "\ref[U]" //Add the master as a faction, allowing inter-mob cooperation
+		if(!iscultist(U))
+			S.mind.store_memory("<b>Serve [U.real_name], your creator.</b>")
 	if(U && iscultist(U))

```

Comment:
You can fold this `if(U && iscultist(U))` into the above `if(U)` check to be just `if(iscultist(U))`.
Same for multiple other checks below.

Reply:
Got it will adjust.